app/src/main/java/org/p2p/wallet/send/ui/main/SendPresenter.kt

```diff
@@ -152,6 +152,14 @@ class SendPresenter(
     override fun setTargetResult(result: SearchResult?) {
         target = result
 
+        result?.let {
+            selectNetworkType(result.networkType)
+            if (!token.isRenBTC && result.networkType == NetworkType.BITCOIN) {
```

Comment:
first statement is redundant cause you check   `if (!token.isRenBTC) return` in `selectNetworkType`